src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -20,10 +21,13 @@ public boolean matchesMysqlType(int type) {
 		}
 	}
 
+	final private boolean isTimestamp = !getType().equals("datetime");
```

Comment:
Probably a dumb question but why not `boolean isTimestamp = getType().equals("timestamp")`?

Reply:
not a dumb question, I was following the previous logic in `matchesMysqlType` but I'll change